plasmapy/constants/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Physical and mathematical constants for use within PlasmaPy"""
+
+from numpy import pi
+
+from astropy.constants.si import h, hbar, k_B, c, G, g0, m_p, m_n, m_e, \
```

Comment:
Most codebases I have seen use brackets to line-break long imports, I think it looks a little nicer.